lib/sinon/assert.js

```diff
@@ -43,6 +43,21 @@ function verifyIsStub() {
     }
 }
 
+function verifyIsValidAssertion(assertionMethod, assertionArgs) {
+    switch (assertionMethod) {
+        case "calledOnce":
```

Comment:
Should this include `notCalled` and `called` as well? http://sinonjs.org/docs/#assert-pass